tls.cipher-suites.config

```diff
@@ -0,0 +1,326 @@
+0x0000,TLS_NULL_WITH_NULL_NULL
```

Comment:
it's simple, so that part I like. I wonder if using yaml would be more consistent and generally nicer? Might lead to easier parsing code too.

@thus  @jasonish what do you think?